src/clarity-angular/data/tree-view/tree-view.module.ts

```diff
@@ -9,20 +9,20 @@ import { CommonModule } from "@angular/common";
 import { ClrIconModule } from "../../icon/icon.module";
 import { ClrFormsModule } from "../../forms/forms.module";
 import { FormsModule } from "@angular/forms";
-import { TREE_VIEW_DIRECTIVES } from "./index";
+import {NEW_TREE_VIEW_DIRECTIVES} from "./index";
 
 @NgModule({
     imports: [
         CommonModule,
         FormsModule, //[(ngModel)] usage in TreeView requires it.
         ClrFormsModule,
-        ClrIconModule,
+        ClrIconModule
     ],
     declarations: [
-        TREE_VIEW_DIRECTIVES
+        NEW_TREE_VIEW_DIRECTIVES
     ],
     exports: [
-        TREE_VIEW_DIRECTIVES
+        NEW_TREE_VIEW_DIRECTIVES
     ]
```

Comment:
This module should export `ClrIfExpandModule`!

If I'm only using the Tree module, I don't want to have to remember to also import `ClrIfExpandModule` separately. I believe you even had the issue yourself in testing. 😂 

This also need to be done for the Datagrid module.